ktor-generator-core/src/main/kotlin/io/ktor/generator/core/features/DefaultFeature.kt

```diff
@@ -0,0 +1,95 @@
+package io.ktor.generator.core.features
+
+/**
+ * Fake feature representing the core of the Ktor itself.
+ *
+ *
+ * Adds the following to Gradle:
+ * - mavenLocal()
+ * - jcenter()
+ * - maven { url 'https://kotlin.bintray.com/ktor' }
+ * - implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+ * - implementation "ch.qos.logback:logback-classic:$logback_version"
+ * - implementation("io.ktor:ktor-server-core:$ktor_version")
+ * - testImplementation "io.ktor:ktor-server-tests:$ktor_version"
+ *
+ * Adds the following to the application code:
+ * - import io.ktor.application.*
+ * - import io.ktor.response.*
+ * - import io.ktor.request.*
+ *
+ * Adds the following to the tests:
+ * - import io.ktor.application.*
+ * - import io.ktor.response.*
+ * - import io.ktor.request.*
+ * */
+object DefaultFeature : KtorFeature {
```

Comment:
I can explain: this is needed for easier template generating. It makes the rocker template code more generic and easy-to-read because I personally think that reading and debugging templates is more complex than chan Kotlin code. So as many of the things to generate easily map to the concept of feature, why not use it?
And this is also the case for some other template generators that I've seen and that have the concept of features.
This just seems more readable and debuggable to me.